packages/zambdas/test/patient-queue.test.ts

```diff
@@ -563,20 +561,20 @@ test('ready for discharge patients queue', () => {
   const { appointments, apptRefToEncounterMap } = getAppointmentsAndMap(visits);
 
   const expectedOrder: Appointment[] = [
-    walkin74MinsAgo.appointment,
-    walkin75MinsAgo.appointment,
-    walkin11MinsAgo.appointment,
-    prebooked15MinEarly.appointment,
-    preBooked17MinEarly.appointment,
-    preBooked5MinEarly.appointment,
-    preBooked10MinEarly.appointment,
+    prebookedRightOnTime.appointment,
+    prebooked15MinEarly2.appointment,
     prebookedAlmostRightOnTime.appointment,
     walkinJustNow.appointment,
-    prebooked15MinEarly2.appointment,
-    prebookedRightOnTime.appointment,
+    preBooked10MinEarly.appointment,
+    preBooked5MinEarly.appointment,
+    preBooked17MinEarly.appointment,
+    prebooked15MinEarly.appointment,
+    walkin11MinsAgo.appointment,
+    walkin75MinsAgo.appointment,
+    walkin74MinsAgo.appointment,
```

Comment:
why not sort these in logical/alphabetical order? eg prebook 17 min early > 15/15 2 > 10 > 5 > almost right on time > right on time, then same for walk in